astroquery/mast/tests/test_mast_remote.py

```diff
@@ -0,0 +1,50 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+from astropy.tests.helper import remote_data
+from astropy.table import Table
+
+from ... import mast
+
+@remote_data
+class TestMast:
```

Comment:
Okay, maybe not for this case. But when people try to subclass it, things are going to break. Better to write `TestMast(object)`. https://wiki.python.org/moin/NewClassVsClassicClass